.azure-pipelines/azure-pipelines-nightly.yml

```diff
@@ -0,0 +1,19 @@
+trigger: none
+pr: none
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight run
+  branches:
+    include:
+    - oe_port
+  always: true
+
+extends:
+  template: template.yml
+  parameters:
+    skip_publish: true
+    test_timeout_minutes: 120 # ethread==1 takes longer to run
```

Comment:
If LTP tests are timing out due to ethread change you might need to increase LTP test timeout as well https://github.com/lsds/sgx-lkl/blob/oe_port/tests/ltp/run_ltp_test.sh#L46

Reply:
Good point, I think the additional timeouts in CI that are still happening are actual deadlocks/hangs, so for now I think 60s per ltp test should work.